charts/logzio-telemetry/values.yaml

```diff
@@ -944,4 +997,61 @@ opencost:
+        custom: container_oom_events_total
+      drop:
+        custom: 
+    # for applications pipeline: applications job
+    applications:
+      keep:
+        custom: avalanche_metric_mmmmm_119_0|avalanche_metric_mmmmm_120_0
+      drop:
+        custom: 
+  
+  # Namespaces names to be filtered
+  namespaces:
```

Comment:
shouldn't this be global filter and not per job?

Reply:
I think its better to keep it separate for flexibility and customization, for example in case a customer wants to scrape application metrics from a specific namespace, but want infrastructure metrics from all namespaces.